homeassistant/components/logger/services.yaml

```diff
@@ -1,6 +1,44 @@
 set_default_level:
   description: Set the default log level for components.
   fields:
-    level: {description: 'Default severity level. Possible values are notset, debug,
-        info, warn, warning, error, fatal, critical', example: debug}
-set_level: {description: Set log level for components.}
+    level:
+      { description: "Default severity level. Possible values are notset, debug,
```

Comment:
Although JSON is allowed, and the old file used it, I would prefer that we just use YAML syntax. Like this example 

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/light/services.yaml#L3

Reply:
done